test/testSetup.ts

```diff
@@ -93,46 +97,52 @@ export class TestSetup {
 
     // Remove the project folder, just in case there are stale files there.
     if (fs.existsSync(this.projectFolderPath)) {
-      await utilities.removeFolder(this.projectFolderPath);
-      await utilities.pause(1);
+      utilities.removeFolder(this.projectFolderPath);
     }
 
     // Now create the temp folder.  It should exist but create the folder if it is missing.
     if (!fs.existsSync(this.tempFolderPath)) {
-      await utilities.createFolder(this.tempFolderPath);
-      await utilities.pause(1);
+      utilities.createFolder(this.tempFolderPath);
     }
 
     utilities.log(`${this.testSuiteSuffixName} - ...finished setUpTestingEnvironment()`);
     utilities.log('');
   }
 
-  public async createInitialProject(scratchOrgEdition: string): Promise<void> {
+  /**
+   * @deprecated - this function has been overcome by events, will be removed soon
+   * @param scratchOrgEdition
+   */
+  public async createInitialProject(scratchOrgEdition: utilities.OrgEdition): Promise<void> {
     utilities.log('');
     utilities.log(`${this.testSuiteSuffixName} - Starting createInitialProject()...`);
 
     // If you are not in a VSCode project, the Salesforce extensions are not running
     // Force the CLI integration extension to load before creating the project
-    const workbench = await (await browser.getWorkbench()).wait();
-    await utilities.runCommandFromCommandPrompt(workbench, 'Developer: Show Running Extensions', 5);
-    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Create Project', 5);
+    const workbench = await utilities.getWorkbench();
+    await utilities.showRunningExtensions();
+    const prompt = await workbench.executeQuickPick('SFDX: Create Project');
+    await utilities.waitForQuickPick(prompt, 'Standard', {
+      msg: 'Expected extension salesforcedx-core to be available within 5 seconds',
+      timeout: 5_000
+    });
     await browser.keys(['Escape']);
     await utilities.pause(1);
     await browser.keys(['Escape']);
 
-    // Do not continue until we verify CLI Integration extension is present and running
-    let coreExtensionWasFound = false;
-    do {
-      await utilities.pause(10);
+    const coreIsActive = await utilities.verifyExtensionsAreRunning(
+      utilities
+        .getExtensionsToVerifyActive()
+        .filter((ext) => ext.extensionId === 'salesforcedx-vscode-core')
+    );
+
+    if (!coreIsActive) {
+      fail('Expected core extension to be active after 20 seconds');
+    }
 
-      coreExtensionWasFound = await utilities.findExtensionInRunningExtensionsList(
-        workbench,
-        'salesforcedx-vscode-core'
-      );
-    } while (coreExtensionWasFound === false);
```

Comment:
Can we try deleting all of this?  So we just create a project directly after switching the Running Extensions view and hope for the best, since we're already assuming that the CLI Integration would start within 5 seconds.

Reply:
@daphne-sfdc this is in the function we are going to remove. Shouldn't be used anymore and I marked it as deprecated.